tests/assertions.js

```diff
@@ -57,3 +54,26 @@ export function shouldBeIssuedVc({issuedVc}) {
       'Expected issuer object to have property id');
   }
 }
+
+function _shouldBeCredentialSubject({credentialSubject}) {
+  // credentialSubject should not be null or undefined
+  should.exist(credentialSubject, 'Expected credentialSubject to exist.');
+  // a credentialSubject can be an Array of objects
+  if(Array.isArray(credentialSubject)) {
+    credentialSubject.length.should.be.gt(
+      0,
+      'Expected credentialSubject to make a claim on at least one subject.'
+    );
+    for(const subject of credentialSubject) {
+      subject.should.be.an(
+        'object',
+        'Expected credentialSubject to be an object.'
+      );
+    }
```

Comment:
```suggestion
    }
```
This should be an object with at least ONE property in it. Ideally, the VC that you pass in should be expanded/recompacted using JSON-LD before sending it to this code to make sure it's a valid JSON-LD object. You could do that in a separate PR.

But one thing you should definitely do is ensure that an empty object isn't provided.

Reply:
Addressed here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/28/commits/f0878d3f9ea47b76d27f093208d13de0ff216d37